app/views/users/show.html.erb

```diff
@@ -52,4 +55,17 @@
       </article>
     <% end %>
   </section>
+  <section>
+    <h1>Friendships</h1>
+    <% if current_user.friendships.empty? %>
+      <h3>You are not currently friends with anyone in the system.</h3>
+    <% else %>
+      <h3>You currently have <%= current_user.friendships.count %> friends:</h3>
```

Comment:
`#length` is more efficient than `#count` in ruby, but this isn't a big deal on small projects.